### PR TITLE
Print remaining JSX node types

### PR DESCRIPTION
**Pull request by pieterv**
Also had to update to jest alpha to get snapshot stuff working.

### CODE REVIEW

File: src/printers/jsx.js [hunk 1]
```diff
@@ -77,4 +77,25 @@ export default {
   JSXText: ({node}) => node.value.trim()
     ? Tokens.string(node.value.trim())
     : null,
+
+  JSXMemberExpression: ({node, print}) => [
```
Comment (kyldvs) [inline src/printers/jsx.js]: alphabetize the printers :)

Comment (pieterv) [inline src/printers/jsx.js]: What you dont have a magic formatter for that 👯  ??

File: src/printers/jsx.js [hunk 2]
```diff
@@ -77,4 +77,25 @@ export default {
...
+    print(node.object),
+    Tokens.string('.'),
+    print(node.property),
+  ],
+
+  JSXNamespacedName: ({node, print}) => [
+    print(node.namespace),
+    Tokens.string(':'),
```
Comment (kyldvs) [inline src/printers/jsx.js]: `Tokens.colon()`

Not sure why exactly, but for most punctuation I have helpers for that token specifically

File: src/printers/jsx.js [hunk 3]
```diff
@@ -77,4 +77,25 @@ export default {
   JSXText: ({node}) => node.value.trim()
     ? Tokens.string(node.value.trim())
     : null,
+
+  JSXMemberExpression: ({node, print}) => [
+    print(node.object),
+    Tokens.string('.'),
```
Comment (kyldvs) [inline src/printers/jsx.js]: Tokens.period()

File: src/printers/jsx.js (hunk 1 above)
Comment (kyldvs) [inline src/printers/jsx.js]: can these be computed?

https://github.com/kyldvs/fjs/blob/master/src/printers/expressions.js#L165

File: src/printers/jsx.js (hunk 2 above)
Comment (pieterv) [inline src/printers/jsx.js]: But not for `Tokens.string('>')` :D ?

Comment (kyldvs) [inline src/printers/jsx.js]: Nope, I choose them semi-randomly :)

File: src/printers/jsx.js (hunk 1 above)
Comment (pieterv) [inline src/printers/jsx.js]: Doesn't look like it (ast explorer with babylon6 says no)

**Comment by pieterv** [comment]
Updated :)

**Pull request merged**